Dockerfile.simpler

```diff
@@ -0,0 +1,11 @@
+FROM docker.io/ubuntu:rolling
+WORKDIR /build
+RUN apt-get update && \
+    apt-get install --no-install-recommends -y \ &&
```

Comment:
it looks like the `&&` is after the `\`.  It should be moved to the new line

Reply:
there should be no `&&` at all here, actually.